gm4-polyfill.js

```diff
@@ -52,14 +52,18 @@ if (typeof GM_registerMenuCommand == 'undefined') {
       console.error('GM_registerMenuCommand got no body.');
       return;
     }
-    let menu = document.getElementById('gm-registered-menu');
+    let menu = null;
+    if (document.body.getAttribute('contextmenu')) {
+        menu = document.querySelector('menu#'+document.body.getAttribute('contextmenu'));
+    }
     if (!menu) {
       menu = document.createElement('menu')
       menu.setAttribute('id', 'gm-registered-menu');
       menu.setAttribute('type', 'context');
       document.body.appendChild(menu);
       document.body.setAttribute('contextmenu', 'gm-registered-menu');
     }
+    document.body.setAttribute('contextmenu', menu.getAttribute('id'));
```

Comment:
This line is redundant, because if another menu already exists it already has the `contextmenu`-attribute, and if a menu is absent the attribute is already set on line 64.